inc/API/preload.php

```diff
@@ -91,6 +91,12 @@ function do_admin_post_rocket_preload_cache() { // phpcs:ignore WordPress.Naming
 		die();
 	}
 
+	$prewarmup_stats = get_option( 'wp_rocket_prewarmup_stats' );
+	if ( empty( $prewarmup_stats['allow_optimization'] ) ) {
```

Comment:
We also need to check for the value of `remove_unused_css` here, else we would bail out all the time, even if the option is not enabled

Reply:
Done